ecs-init/docker/docker_test.go

```diff
@@ -247,6 +251,20 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 	if certDir := config.HostPKIDirPath(); certDir == "" {
 		expectedAgentBinds = expectedAgentBindsSuseUbuntuPlatform
 	}
+	// add exec binds
+	hostCapabilityExecResourcesDir := filepath.Join(hostCapabilitiesResourcesRootDir, capabilityExecName)
+	bindSourcePaths := []struct {
+		path      string
+		predicate func(os.FileInfo) bool
+	}{
+		{filepath.Join(hostCapabilityExecResourcesDir, capabilityExecHostBinRelativePath), isDir},
+		{filepath.Join(capabilityExecHostCertsDir, capabilityExecRequiredCert), isFile},
+	}
+	for _, path := range bindSourcePaths {
+		if existsAndValid, err := pathExistsAndValid(path.path, path.predicate); err == nil && existsAndValid {
```

Comment:
not handling `err` here since `pathExistsAndValid` is already called in `docker.go`, errors should have been thrown there